src/mantarray_desktop_app/mc_comm.py

```diff
@@ -412,7 +425,14 @@ def _process_main_module_comm(self, comm_from_instrument: bytes) -> None:
                 )
             prev_command = self._commands_awaiting_response.popleft()
             if prev_command["command"] == "handshake":
-                # see note above: Tanner (3/17/21)
+                status_code = int.from_bytes(response_data, byteorder="little")
+                log_msg = f"Handshake response received from instrument. Status Code: {status_code}"
```

Comment:
note for future---lines 408-419 seem substantively similar...could consider extracting out as a common function